etc/JDownloader.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for JDownloader
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/JDownloader.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# generic gui profile
+# depending on your usage, you can enable some of the commands below:
+
+include /etc/firejail/disable-common.inc
+# include /etc/firejail/disable-devel.inc
```

Comment:
any specific reason disable-devel.inc is commented?

Reply:
your suggestion for adding the noblacklist options for java fixed the issues i had with that - uncommented.